accounts-db/src/accounts_db/tests.rs

```diff
@@ -5984,16 +6006,14 @@ pub(crate) fn compare_all_accounts(
 fn test_shrink_ancient_overflow_with_min_size() {
     solana_logger::setup();
 
-    let ideal_av_size = ancient_append_vecs::get_ancient_append_vec_capacity();
     let num_normal_slots = 2;
 
     // build an ancient append vec at slot 'ancient_slot' with one `fat`
     // account that's larger than the ideal size of ancient append vec to
     // simulate the *oversized* append vec for shrinking.
-    let account_size = (1.5 * ideal_av_size as f64) as u64;
```

Comment:
Note: I think this test and `test_shrink_overflow_too_much` are actually not testing what the comments are saying, since lowering the account size below ideal ancient storage size won't make the accounts packed into the first slot.
I would like to address this in separate PR, though having trouble following the logic by how accounts are selected to be shrunk...

Reply:
this test passes for almost any value picked here for `account_size` (or `ideal_av_size` as basis for account_size), since for this test data `combine_ancient_slots_packed` won't ever shrink the storages (I think there were significant changes in how we pick storages to shrink since this test was written and even adjusting ` `PackedAncientStorageTuning` to be more aggressive won't cause it).

previously I had it as `ancient_append_vecs::get_ancient_append_vec_capacity() / 100`, either way at this time this tests seems to not really test anything